README.md

```diff
@@ -65,7 +65,7 @@ Always be considerate and respectful, specifically:
 
 Instead:
 
-- ask questions, assuming that everyone is intelligent and well-meaning
```

Comment:
Can `intelligent` be an exclusive word?

Reply:
How can it be if you assume that _everyone_ is? :)